packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -841,18 +843,10 @@ const useSingleSiteCommands = ( { navigate, currentRoute }: useCommandsParams ):
 		{
 			name: 'openWooCommerceSettings',
 			label: __( 'Open WooCommerce settings', __i18n_text_domain__ ),
```

Comment:
[random idea, not blocking in the slightest] it might be cool if the name changed between 'Open WooCommerce settings' and 'Install WooCommerce' depending on whether or not it was already installed.

Reply:
We would need to change that on the normal command too. Let's revisit once everything is launched.